src/main/java/hu/oe/nik/szfmv/automatedcar/visualization/Dashboard.java

```diff
@@ -8,25 +18,75 @@
  */
 public class Dashboard extends JPanel {
 
-    private final int width = 250;
-    private final int height = 700;
-    private final int backgroundColor = 0x888888;
+    private static final Logger LOGGER = LogManager.getLogger();
+
+    private static final int WIDTH = 250;
+    private static final int HEIGHT = 700;
+    private static final int BACKGROUND_COLOR = 0x888888;
+
+    private Gui parent;
+
+    private TurnSignal leftTurnSignal;
+    private TurnSignal rightTurnSignal;
+    private TextSignal accSpeedSignal;
+    private TextSignal accDistanceSignal;
+    private TextSignal accOnOffSignal;
+    private TextSignal ppSignal;
+    private TextSignal lkaSignal;
+    private TextSignal lkaWarningSignal;
+    private TextSignal trafficSignSignal;
+    private TextSignal aebWarningSignal;
+    private TextSignal rrWarningSignal;
+    private DashboardText currentGear;
+    private DashboardText speedLimit;
+    private DashboardText steeringWheel;
+    private DashboardText xCoordinate;
+    private DashboardText yCoordinate;
+    private ProgressBars gasProgressBar;
+    private ProgressBars breakProgressBar;
+    private Gauge rpmGauge;
+    private Gauge kmhGauge;
 
-    Gui parent;
 
     private Thread timer = new Thread() {
-        int difference;
+        //int difference;    // ez nem tudom mire valo, egyelore maradjon
 
         public void run() {
             while (true) {
-
                 try {
-                    Thread.sleep(100);
+                    VirtualFunctionBus vfb = parent.getVirtualFunctionBus();
+                    if (vfb != null) {
+                        handleInputPacket(vfb.inputPacket);
+
+                        //...
```

Comment:
Ez itt mi?

Reply:
Csak egy jelzés hogy a többi packet-et is le kell itt kezelni. Csak még nem jutottunk odáig hogy a másik csapat kódjával integráljunk. Ha jól gondolom ahhoz először a master-be kellene kerülnie azon kódoknak, amit itt használnánk a másik csapattól (a packet osztály, és elérése a VFB-ről). Vagy rosszul gondolom?

Reply:
A ... Nekem ezt nem tette egyértelművé.